src/components/Tooltip/index.js

```diff
@@ -387,7 +392,8 @@ Tooltip.defaultProps = {
   },
   spaceFromMouse: SPACE_FROM_MOUSE,
   reduceTop: 0,
-  reduceBottom: 0
+  reduceBottom: 0,
+  preferTop: false
 };
```

Comment:
Are you going to add another prop like onDirectionChange to let the caller adjust the display?

Reply:
this is mandatory in order to make the integration work